### ISSUE TITLE
The output results are in wrong percentages

### ISSUE THREAD

**Issue by raytsangGitHub** [raytsangGitHub/MyPortfolio-AiApps#3]
**Describe the bug**
Compile the console app, the results are in wrong percentages:
1. "Sealyham terrier, Sealyham" (1,353.90%)
  2. "golden retriever" (1,319.47%)
  3. "Tibetan terrier, chrysanthemum dog" (1,260.13%)
  4. "Labrador retriever" (1,128.76%)
  5. "Afghan hound, Afghan" (1,124.25%)

**To Reproduce**
Run the application.

**Expected behavior**
 1. "Sealyham terrier, Sealyham" (36.35%)
  2. "golden retriever" (25.76%)
  3. "Tibetan terrier, chrysanthemum dog" (14.23%)
  4. "Labrador retriever" (3.83%)
  5. "Afghan hound, Afghan" (3.66%)

**Comment by raytsangGitHub** [comment]
The original result was in true probabilities format, To fix by apply softmax to the output array.

1. Create the static function. static float[] Softmax(float[] logits)
{
    var maxLogit = logits.Max(); // for numerical stability
    var exps = logits.Select(v => MathF.Exp(v - maxLogit)).ToArray();
    var sumExps = exps.Sum();
    return exps.Select(e => e / sumExps).ToArray();
}

2. Pass the array to this function. 
var rawOutput = results.First(v => v.Name == outputName).AsEnumerable<float>().ToArray();
var scores= Softmax(Output);
